fullnode_install.sh

```diff
@@ -123,8 +123,8 @@ function init_centos(){
         exit 0
     fi
 
-    echo "Installing Ansible, net-tools and git..."
-    yum install ansible git net-tools -y
+    echo "Installing Ansible, git and pv..."
+    yum install ansible git pv -y
```

Comment:
Actually, here we're only installing what is required to run the playbook.
All other dependencies are installed in the playbook. There are users who don't use this script. They clone and run the playbook. That's why its better to add it here for yum:
https://github.com/nuriel77/iri-playbook/blob/master/roles/iri/tasks/deps_yum.yml#L42

Reply:
Okay, added "pv" also to the playbook.